### PR TITLE
[13.0] [MIG] sale_commission_pricelist

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

File: sale_commission_pricelist/models/sale_order.py [hunk 1]
```diff
@@ -26,13 +26,12 @@ def _get_commission_from_pricelist(self):
     def _onchange_product_id_sale_commission_pricelist(self):
```
Comment (pedrobaeza) [inline sale_commission_pricelist/models/sale_order.py]: This should be replaced by a new `depends` on the compute field of `agent_ids` to `order_id.pricelist_id`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please squash both migration commits (the pylint one and the real migration).

File: sale_commission_pricelist/models/sale_order.py (hunk 1 above)
Comment (feketemihai) [reply sale_commission_pricelist/models/sale_order.py]: @pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

**Comment by feketemihai** [comment]
@pedrobaeza Merge 13.0 again, so no conflict.

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**